plugins/woocommerce-admin/client/products/layout/product-field-layout.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import './product-field-layout.scss';
+
+export const ProductFieldLayout: React.FC = ( { children } ) => {
+	return <div className="product-field-layout">{ children }</div>;
```

Comment:
A slotfill could be added here after rendering the children so 3PD's can render fields within a specific section.